docs/tvl.md

```diff
@@ -13,19 +13,32 @@ As you can see there are three types of value:
 Everything you see here is achieved via proper configuration files in our repository.
 
 ## Token guide
-
-1. Go to `config\src\tokens\source.jsonc`
+1. Go to `config\src\tokens\tokens.jsonc`
 2. Add your token to the proper chain
 
-   - required fields are: **symbol & address**
-   - optional fileds are: type, formula, category, bridgedUsing & coingeckoId
-   - for `ethereum` the `type` and `formula` are `CBV` and `locked` by default, for other chains you need to specify it
-   - `category` is `other` by default
-   - if you do not see your chain run `yarn coingecko:platforms` (TODO: update docs when dynamic chains functionality is merged)
+→ Canonically bridged tokens (CBV) should always be added to Ethereum.
+- The required fields are `symbol` **&** `address`.
+- Optional fields are: `type`, `formula`, `category`, `bridgedUsing` & `coingeckoId`.
+- By default, the `type` is `CBV`, the `formula` is `locked`, and the `category` is `other`.
 
+→ Externally bridged tokens (EBV) and natively minted tokens (NMV) should be added to the token list of a specific L2.
+- If the chain is missing in the file, you have to add it
+- The required fields are symbol, address, type, formula. The optional fields are `category`, `bridgedUsing` & `coingeckoId`.
+    - If the token is externally bridged, you have to add a `bridgedUsing` object, containing at least the `bridge` = the bridge name. If we have it on our website, you can also add a `slug`.
+    - Make sure to run `yarn coingecko:platforms`. If the chain is not available there, you need to also specify the `coingeckoId`.
```

Comment:
The inconsistent indentation here messes up the list. Ideally all numbered items (`1.`) wouldn't be indented and all nested items (`-`) would be indented by 3 spaces.